artio-core/src/main/java/uk/co/real_logic/artio/engine/framer/FramerContext.java

```diff
@@ -86,9 +86,11 @@ public FramerContext(
             outboundLibraryStreams.dataPublication("outboundPublication"));
 
         sentSequenceNumberIndex = new SequenceNumberIndexReader(
-            configuration.sentSequenceNumberBuffer(), errorHandler, configuration.logFileDir());
+            configuration.sentSequenceNumberBuffer(), errorHandler, recordingCoordinator.outboundRecordingIdLookup(),
```

Comment:
These changes now mean that the RecordingIdLookup class can be used on the Framer and Indexer agents. It's not a class that is designed to be threadsafe. Probably best to pass out separate instances to each thread.

Reply:
fixed, thanks!